app/helpers/govuk_form_helper.rb

```diff
@@ -20,7 +20,8 @@ def form_group(key, &b)
 
     def label(*args)
       options = args.extract_options!
-      args << options.merge(class: "govuk-label")
+      options[:class] = "govuk-label " + options[:class].to_s
```

Comment:
do you need the `#to_s`?

Reply:
yes cause it may be nil

Reply:
how about replace it with `options.fetch(:class, "")` so as to make it more clear?

Reply:
yess that's definitely better

Reply:
done